packages/analytics-js-common/src/types/ErrorHandler.ts

```diff
@@ -19,3 +21,5 @@ export interface IErrorHandler {
   notifyError(error: Error): void;
   attachErrorListeners(): void;
 }
+
+export type PreloadedError = Array<any>;
```

Comment:
Why is it of `any` type? We are aware of the data types.

Reply:
Yes, thank you I missed updating this